mailing/src/main/java/org/lukawska/trainSmart/mailing/application/exception/RestException.java

```diff
@@ -0,0 +1,14 @@
+package org.lukawska.trainSmart.mailing.application.exception;
+
+import lombok.Getter;
+
+@Getter
+public class RestException extends RuntimeException {
```

Comment:
nie pasuje mi ta nazwa. Restowe wyjatki to juz spring sam w sobie obsluguje. I pomysl czy moze da sie extendowac cos innego niz runtime w tej sytuacji